src/Components/Color/Color.jsx

```diff
@@ -2,7 +2,15 @@ import "./Color.css";
 
 /* functional component named ColorCard */
 /* receives a single prop called color, which is an object */
-export default function ColorCard({ color }) {
+export default function ColorCard({ color, onDelete }) {
+  const handleDeleteClick = () => {
+    const isConfirmed = window.confirm("Really delete?");
```

Comment:
nice!

Reply:
you gave me the hint thanks!, I looked into Predrags comments.